src/Resources/config/app/fixtures.yaml

```
@@ -41,11 +46,11 @@ sylius_fixtures:
                 payment_method:
                     options:
                         custom:
-                            global_quickpay:
-                                code: "global_quickpay"
-                                name: "QuickPay Global"
-                                instructions: "Pay via QuickPay with credit card or PayPal"
-                                gatewayName: quickpay
+                            cc_quickpay:
+                                code: 'cc_quickpay'
```

```suggestion
                                code: 'quickpay_credit_card'
```

To make it easier to understand just by seeing it